### PR TITLE
 The solution for insure #29 Inline JavaScript <script> tag parse error 

### PR DESCRIPTION
**Pull request by ranforce**
This pull request is The solution for insure #29 Inline JavaScript <script> tag parse error 
The most important change :
https://github.com/izolate/html2pug/pull/30/files#diff-6947033678b93d106e25614dd972e66fR100

Thanks

### CODE REVIEW

File: src/parser.js [hunk 1]
```diff
@@ -81,7 +95,12 @@ class Parser {
       let line = `${indent}${this.setAttributes(node)}`
 
       if (this.isUniqueTextNode(node)) {
-        line += ` ${node.childNodes[0].value}`
+        let innerText = node.childNodes[0].value
+        if (node.tagName === 'script') {
+          line += `.\n${this.parseJavascriptString(innerText, indent)}`
```
Comment (ranforce) [inline src/parser.js:100]: This line to parse inline script tag

**Comment by JPeer264** [comment]
@ranforce neat, thanks for your PR. I ran into the same issue.

@izolate any chance, this gets a review and can get merged?

**Comment by JPeer264** [comment]
@ranforce is it possible that you can add this for `style` as well?

**Comment by ranforce** [comment]
@JPeer264  In my case, i didn't get more changes to use this pkg, u can try to fork , publish, and use ur own pkg.

**Comment by JPeer264** [comment]
@ranforce thanks for the follow up, the update and your suggestions :+1:

**Comment by izolate** [comment]
Hi @ranforce ,

Thank you for your contributions, but I'm closing this in favour of #35. The library core has been refactored and these issues addressed in the linked PR.

cc @JPeer264

**Pull request closed without merging**